internal/character/silverwolf/technique.go

```diff
@@ -0,0 +1,25 @@
+package silverwolf
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+func (c *char) Technique(target key.TargetID, state info.ActionState) {
+	c.engine.Attack(info.Attack{
+		Source:     c.id,
+		Targets:    c.engine.Enemies(),
+		DamageType: model.DamageType_QUANTUM,
+		AttackType: model.AttackType_MAZE,
+		BaseDamage: info.DamageMap{
+			model.DamageFormula_BY_ATK: 0.8,
+		},
+		StanceDamage: 0,
+		EnergyGain:   0.0,
+	})
+
+	for _, trg := range c.engine.Enemies() {
+		c.engine.ModifyStance(trg, c.id, 60)
+	}
```

Comment:
New issue with current sim implementation. Cannot trigger a "Quantum Weakness Break" here. I think the missing piece is `ModifyStance` needs to take in a `DamageType` so that can be included in the `BreakEvent`? depends on how break implementation looks

Reply:
I've only observed break being tied directly to the character trigger. So in this case, we could derive the damage type from the included `c.id` parameter (source). Maybe we should look at dm to see if this is a safe work around?

Reply:
Left a note here for future implementation: #24